src/tests/conformance/messages.rs

```diff
@@ -77,7 +75,7 @@ async fn reject_invalid_messages() {
     // List of test messages and their expected Reject kind.
     let cases = vec![
         (
-            Message::Version(Version::new(node.addr(), new_local_addr())),
+            Message::Version(Version::new(node.addr(), "0.0.0.0:0".parse().unwrap())),
```

Comment:
This is equivalent to the previous code, just removed the helper. We may want to include a different address here in future but it will require refactoring the test.